google-cloud-storage/src/main/java/com/google/cloud/storage/BlobReadChannel.java

```diff
@@ -158,6 +164,13 @@ public int read(ByteBuffer byteBuffer) throws IOException {
     return toWrite;
   }
 
+  @Override
+  public LimitableReadChannel limit(long limit) {
```

Comment:
I was thinking you could overload read() but that interface comes from ReadableByteChannel.

What about introducing a new option into requestOptions to define range?